buildSrc/out.gradle

```diff
@@ -25,7 +25,8 @@ def chooseOutDir(subdir = "") {
     if (outDir == null) {
         def checkoutRoot = System.getProperty("CHECKOUT_ROOT")
         if (checkoutRoot == null) {
-            checkoutRoot = new File("${buildscript.sourceFile.parent}/../../..")
+            def workingDir = new File(buildscript.sourceFile.parent)
```

Comment:
not sure how this works. git repo is the same in the repo tool checkout as well.
we might consider hardcoding parent folders and cheeck for frameworks/support

Reply:
Ah yeah I may have mispoke here, the git repo root is the same, but this returns the repo checkout root as its relative of buildSrc/ but this leads outside the checkout in github